data/json/items/comestibles/drink.json

```diff
@@ -275,7 +275,7 @@
     "name": { "str_sp": "sweetened coffee substitute with milk" },
     "calories": 88,
     "delete": { "flags": [ "NUTRIENT_OVERRIDE" ] },
-    "description": "The morning ritual of the post-apocalyptic New England.  Brewed from the Kentucky coffeetree, this drink has no actual caffeine, but serves as a replacement for those who dearly miss the aroma of real coffee.",
+    "description": "The morning ritual of the post-apocalyptic world.  Brewed from the Kentucky coffeetree, this drink has no actual caffeine, but serves as a replacement for those who dearly miss the aroma of real coffee.",
```

Comment:
The game takes place in New England it made sense to reference that I think

Reply:
The change is to better match the description of regular black coffee: "The morning ritual of the pre-apocalyptic world". I won't die on the hill for this one though, if you think I should revert it I will.